recordium/config.py

```diff
@@ -53,11 +53,14 @@ def __getattr__(self, key):
 
     def __setattr__(self, key, value):
         if key in self._config_options:
-            if self.data.get(key, self._config_options[key]) != value:
+            if self.data.get(key) != value:
                 self.data[key] = value
                 self._need_save += 1
         else:
-            super(_Config, self).__setattr__(key, value)
+            if key.upper() != key:
```

Comment:
What is this .upper() here? Why are you checking if the attribute is uppercase? using lowercase should behave the same than using a non-existant uppercase key, and it should raise AttributeError.